plasmalogin-helper[45559]: PAM _pam_load_conf_file: unable to open config for system-login
plasmalogin-helper[45559]: PAM _pam_load_conf_file: unable to open config for system-login
plasmalogin-helper[45559]: PAM _pam_load_conf_file: unable to open config for system-login
plasmalogin-helper[45559]: PAM _pam_load_conf_file: unable to open config for system-login
plasmalogin-helper[45559]: [PAM] Starting...
plasmalogin-helper[45559]: [PAM] Authenticating...
plasmalogin-helper[45559]: [PAM] authenticate: Permission denied
plasmalogin-helper[45559]: [PAM] returning.
plasmalogin-helper[45559]: [PAM] Asked to close the session but it wasn't previously open
plasmalogin-helper[45559]: [PAM] Ended.

--- plasma-login-manager/data/pam/arch/plasmalogin-autologin.vanilla	2026-01-04 20:04:16.377435033 +0100
+++ plasma-login-manager/data/pam/arch/plasmalogin-autologin	2026-01-04 20:05:24.166674815 +0100
@@ -10,8 +10,8 @@
 auth        required    pam_permit.so
 -auth       optional    pam_gnome_keyring.so
 -auth       optional    pam_kwallet5.so
-account     include     system-local-login
-password    include     system-local-login
-session     include     system-local-login
+account     include     local-login
+password    include     local-login
+session     include     local-login
 -session    optional    pam_gnome_keyring.so auto_start
 -session    optional    pam_kwallet5.so auto_start
--- plasma-login-manager/data/pam/arch/plasmalogin.vanilla	2026-01-04 20:04:16.372016651 +0100
+++ plasma-login-manager/data/pam/arch/plasmalogin	2026-01-04 20:04:29.729476221 +0100
@@ -3,16 +3,16 @@
 # SPDX-License-Identifier: CC0-1.0
 # SPDX-FileCopyrightText: none
 
-auth        include     system-login
+auth        include     login
 -auth       optional    pam_gnome_keyring.so
 -auth       optional    pam_kwallet5.so
 
-account     include     system-login
+account     include     login
 
-password    include     system-login
+password    include     login
 -password   optional    pam_gnome_keyring.so    use_authtok
 
 session     optional    pam_keyinit.so          force revoke
-session     include     system-login
+session     include     login
 -session    optional    pam_gnome_keyring.so    auto_start
 -session    optional    pam_kwallet5.so         auto_start
